tests/test.sh

```diff
@@ -364,17 +369,29 @@ run_ice()
         fi
     fi
 
-    remove_debug_info="s/DW_AT_\(GNU_dwo_\(id\|name\)\|comp_dir\|producer\|linkage_name\|name\).*//g"
+    local remove_offset_number='s/<[A-Fa-f0-9]*>/<>/g'
+    local remove_debug_info="s/\(Length\|DW_AT_\(GNU_dwo_\(id\|name\)\|comp_dir\|producer\|linkage_name\|name\)\).*/\1/g"
+    local remove_debug_pubnames='/Offset\s*Name/{n;s/\( *\)[A-Fa-f0-9]*\(\s*.*\)/\1\t\2/}'
```

Comment:
g++ 4.8.4 appears to be using a .debug_pubnames section where the newer version uses a .debug_gnu_pubnames section. The gnu version will always have the same "Size of area" value and offset number for the function where the debug_pubnames version will generate different values for there (this happens even when if there are no changes in environment, just running the g++ command twice will produce different output here).